backend/codesirius/problems/serializers/reference_solution.py

```
@@ -0,0 +1,62 @@
+            )
+            return ReferenceSolution.objects.create(
+                **validated_data, problem=self.context["problem"]
+            )
+        except Exception as e:
+            logger.error(f"Failed to create a new reference solution: {e}")
+            raise serializers.ValidationError(
+                "Failed to create a new reference solution"
+            )
+
+    def update(
+        self, instance: ReferenceSolution, validated_data: Dict
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)